guides/configuration-types.rst

```diff
@@ -28,7 +28,7 @@ names <https://venus.cs.qc.cuny.edu/~krishna/cs111/lectures/D3_C++_Variables.pdf
 
 -  … must start with a letter and can end with numbers.
 -  … must not have a space in the name.
--  … can not have special characters except the underscore (“_“).
+-  … can not have special characters except underscore (“_“) and hyphen ("-").
```

Comment:
Uhm no this line should stay. This section is about `id` names, so the names of variables. C++ does not allow hyphens there.

Reply:
Roger, I got trigger-happy searching for underscore.  Fixed now.